.ci/docker.sh

```diff
@@ -43,7 +47,7 @@ while [[ $# != 0 ]]; do
     *)
       if [[ $1 == -* ]]; then
         echo "unrecognized option: $1"
-        return 3
+        exit 3
```

Comment:
no, the interactive option is just extra, the normal use is to source the script

Reply:
Fixed